### ISSUE TITLE
Storage resources are being removed by resource manager during format of ldiskfs devices

### ISSUE THREAD

**Issue by tanabarr** [intel-hpdd/intel-manager-for-lustre#491]
During SSI test run http://jenkins.lotus.hpdd.lab.intel.com/job/integration-tests-shared-storage-configuration/4088/ on the reactive architecture patch stack (https://github.com/intel-hpdd/intel-manager-for-lustre/pull/467) when creating filesystem, failure occurs because disk[12,14] missing a single `VolumeNode`  each (only 6 in db where 8 are expected). 

```
chroma=> select * from chroma_core_volumenode where not_deleted=true;
 id | volume_id | host_id |                          path                          | storage_resource_id | primary | use | not_deleted
----+-----------+---------+--------------------------------------------------------+---------------------+---------+-----+-------------
  1 |         4 |       1 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk14        |                  13 | f       | f   | t
 23 |         4 |       2 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk14        |                  78 | f       | f   | t
  2 |         5 |       1 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk12        |                  18 | f       | f   | t
 24 |         5 |       2 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk12        |                  83 | f       | f   | t
...
 34 |         4 |       4 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk14        |                 107 | f       | t   | t
 13 |         5 |       3 | /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_disk12        |                  59 | f       | t   | t
```

The reason for the missing  nodes is not, as was previously hypothesised, due to exceptions when processing the `device-aggregator` output which was, in turn, caused by differences in formatting of the host data (https://github.com/intel-hpdd/device-aggregator/issues/16).

Another possible cause is interaction between `lustre` and `linux` plug-ins, which results in primary volumenodes being removed when targets are formatted. The cause presumably exists because of a subtle difference in how the `linux` plug-in now uses the aggregate output on the manager side.

The question is why is the resource manager removing the underlying storage resources on format.

**Comment by tanabarr** [comment]
maybe changes to the UpdateManagedTargetMount code could have had an effect?

**Comment by tanabarr** [comment]
This issue is a result of lustre ldiskfs formatted disks (reported by udev as ext4), being identified as `LocalMount`s which results in the relevant `VolumeNode`s being removed. 

Temporary fix is to not identify all ext4 formatted disks as local mounts, permanent fix is to move local Mount identification and processing into the `device-scanner`.

**Comment by tanabarr** [comment]
fix merged into #458
